drgn/commands/_builtin/crash/ps.py

```diff
@@ -0,0 +1,294 @@
+        print(
+            f"[{last_arrival:>14}] [{state}]  PID: {pid:<8} TASK: {t.value_():x}   CPU: {cpu:<2}  COMMAND: {command}"
+        )
+
+
+def show_tasks_last_runtime_timestamp(tasks: List[Object]) -> None:
+    """
+    Display task information sorted by last_run/timestamp/last_arrival, showing the raw timestamp.
+    """
+    tasks.sort(key=get_task_arrival_time, reverse=True)
+    for t in tasks:
+        cpu = task_cpu(t)
+        pid = t.pid.value_()
+        state = task_state_to_char(t)
+        command = double_quote_ascii_string(t.comm.string_())
+        last_arrival = get_task_arrival_time(t)
+        print(
+            f"[{last_arrival:>14}] [{state}]  PID: {pid:<8} TASK: {t.value_():x}   CPU: {cpu:<2}  COMMAND: {command}"
+        )
+
+
+def show_taskinfo(prog: Program, tasks: List[Object]) -> None:
+    """
+    Display tasks informations.
+    """
+    rows: List[Sequence[Any]] = [
+        (
+            CellFormat("PID", ">"),
+            CellFormat("PPID", ">"),
+            CellFormat("CPU", "^"),
+            CellFormat("TASK", "^"),
+            CellFormat("ST", ">"),
+            CellFormat("%MEM", ">"),
+            CellFormat("%VSZ", ">"),
+            CellFormat("RSS", ">"),
+            CellFormat("COMM", "<"),
+        )
+    ]
+    tasks.sort(key=lambda t: t.pid.value_())
+    page_size = int(prog["PAGE_SIZE"])
+    total_mem = totalram_pages(prog)
+    for t in tasks:
+        task_rss = get_task_rss_info(prog, t)
+        rss_kb = task_rss.total * page_size // 1024
+        pct_mem: float = task_rss.total / total_mem
+        rows.append(
+            [
+                t.pid.value_(),
+                t.parent.pid.value_(),
+                CellFormat(task_cpu(t), "^"),
+                CellFormat(t.value_(), "^x"),
+                task_state_to_char(t),
+                CellFormat(pct_mem, ".1%"),
+                task_vsize(t) // 1024,
+                rss_kb,
+                (
+                    f"[{escape_ascii_string(t.comm.string_())}]"
+                    if task_is_kthread(t)
+                    else f"{escape_ascii_string(t.comm.string_())}"
+                ),
+            ]
+        )
+    print_table(rows)
+
+
+def check_arg_type(arg: Optional[str]) -> Tuple[str, Any]:
+    """
+    Check the filter type of the argument
+    """
+    if arg is not None:
+        try:
+            return ("pid", int(arg, 10))
+        except ValueError:
+            pass
+        try:
+            return ("task", int(arg, 16))
+        except ValueError:
+            return ("comm", arg)
+    else:
+        return ("none", None)
+
+
+@crash_command(
+    description="process information",
+    long_description="display process status information",
+    arguments=(
+        mutually_exclusive_group(
+            argument(
+                "-u",
+                dest="user",
+                action="store_true",
+                default=False,
```

Comment:
`default` is automatically `False` when `action="store_true"`, so you can drop this (and all of the other ones).
```suggestion
```